src/views/collections/CollectionsExplore/CollectionsExplore.vue

```diff
@@ -0,0 +1,228 @@
+<template>
+  <EternaPage    v-if="fetchState.firstFetchComplete && collections"
+>
+      <h3 :style="{ fontSize: '16px', fontWeight: 'bold' }">
+        {{ $t('collections-view:top-tip') }}
+      </h3>
+
+      <QuestCarousel :slideTo="slideTo">
+        <SwiperSlide v-for="item in quests" :key="item.name">
+          <CollectionCard :key="item.name" :cleared="cleared" v-bind="item" />
+        </SwiperSlide>
+      </QuestCarousel>
+
+      <h4
+        :style="{
+          fontSize: '16px',
+          fontWeight: 'bold',
+          textTransform: 'uppercase',
```

Comment:
Why is this subtitle uppercased and the other isn't? I would keep them both standard.